content/docs/git/ide.md

```diff
@@ -0,0 +1,248 @@
+- Stage the changes by running `git add .` or `git add --all` and check the status again to make sure the changes have shown up in your staging area. They should be green
+- Create a commit now to "save" your changes--in this case, the deleted file. Try to leave a helpful commit message like below, replacing <name-of-file> with the file you deleted:
+
+  ```sh
+  git commit -m "Created my first journal!"
+  ```
+
+- Check that the commit was successful by checking the status of the repository (`git status`) and checking the log (`git log`). You should see the commit message that you added at the top of the log, in addition to some previous commits from before you cloned the repository.
+
+- Let's go ahead and push these commits so far to the remote repository. Run the following, replacing `<current-branch>` with the branch you are working on (if you forget what it is called, you can run `git status` or `git branch` and copy the branch name from there!):
+
+```sh
+git push origin <current-branch>
+```
+
+### 6. Create a pull request (PR) on GitHub
+
+As we did at the end of last class, let's create a pull request to merge this `test-branch` to `main`.
+
+Go to GitHub and view your branch: on the left side, there should be a button with `main` and a down arrow -- click this and select your branch. At the top, if there is an option to "Pull request", click that, otherwise click on the "Pull request" tab and then "New pull request".
+
+Since this is a forked branch, we are given the option of submitting a pull request against the original repository that we forked from (listed under `base repository`). This is often how you can contribute to open source projects -- by forking the repository, making your changes, and then submitting a PR against the original repo for the maintainers of the project to accept or reject.
+
+For now, let's just PR within the fork, so change the `base repository` to be the forked repo. (The `base repository` and `head repository` should match, but the `base` branch should be `main`).
+
+Assign someone in your group to the PR 'it can be seen in the right section and approve and merge the changes into `main`.
+
+## 7. (Optional) Let´s swap places
+
+Now it is time for your partner to repeat what you have done. 
+
+BUT! instead of forking the repository we will **clone directly your partner repositor**y so both of you can contribute towards a shared journal.
+
+For that let´s do the following:
+- Ask your partner for an URL pointing to his journal repository that both of you have created and existts in Github.
+- Let´s **clone** the repository (make sure the Pull Request opened is merged and that the main branch has the content you just created)
+- Add some new files, folders, images and after add, commit and push your changes. You can go back to section 3 and repeat the same instructions
+
+## Extra resources
```

Comment:
optional: could also add a link to the official VSCode tutorials/docs: https://code.visualstudio.com/docs/introvideos/basics

Reply:
Good suggestion!! 🎉